packages/library-sync/src/commonMain/kotlin/io/realm/mongodb/App.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.mongodb
+
+import io.realm.mongodb.internal.AppConfigurationImpl
+import io.realm.mongodb.internal.AppImpl
+import kotlinx.coroutines.CoroutineDispatcher
+
+/**
+ * TODO
+ */
+interface App {
+
+    val configuration: AppConfiguration
+
+    suspend fun login(credentials: Credentials): Result<User>
```

Comment:
Missing Dokka, or as a minimum a TODO about it.

Reply:
There is no Dokka-documentation in any of the classes yet, so makes most sense to address at the same time. Added a TODO.